### PR TITLE
Add recipe for pywinauto

### PR DESCRIPTION
**Pull request by m-rossi**

### CODE REVIEW

File: recipes/pywinauto/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - pip
+  run:
+    - comtypes
+    - python
+    - pywin32
+    - six
+
+test:
+  imports:
+    - pywinauto
```
Comment (synapticarbors) [inline recipes/pywinauto/meta.yaml]: Please also test the submodules:

```
- pywinauto.controls
- pywinauto.linux
```

File: recipes/pywinauto/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+build:
+  number: 0
+  skip: True  # [not win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
```
Comment (synapticarbors) [inline recipes/pywinauto/meta.yaml]: It looks like there are different requirements based on the platform:

https://github.com/pywinauto/pywinauto/blob/master/setup.py#L68

File: recipes/pywinauto/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  sha256: e115463806acc41b8b5aeaeaece9cd5996a3091d1baa18b1dc91736c7684449d
+
+build:
+  number: 0
+  skip: True  # [not win]
```
Comment (synapticarbors) [inline recipes/pywinauto/meta.yaml]: I'm a little confused as to whether this is a windows only package:

https://github.com/pywinauto/pywinauto/blob/master/pywinauto/linux/clipboard.py#L53

File: recipes/pywinauto/meta.yaml (hunk 1 above)
Comment (m-rossi) [reply recipes/pywinauto/meta.yaml]: Done.

File: recipes/pywinauto/meta.yaml (hunk 2 above)
Comment (m-rossi) [reply recipes/pywinauto/meta.yaml]: I've added them with selectors.

File: recipes/pywinauto/meta.yaml (hunk 3 above)
Comment (m-rossi) [reply recipes/pywinauto/meta.yaml]: Me too, but it seems there has been added basic support for linux. I've adjusted the recipe.

**Pull request merged**